### ISSUE TITLE
Command-clicking a post should open it and its thread in a new tab

### ISSUE THREAD

**Issue by Cassolotl** [mastodon/mastodon#21553]
### Pitch

In the not-advanced theme, when you click a post it opens the post and its replies. But when you command-click on Mac, you expect it to open it in a new tab, but nothing happens.

It should open in a new tab, please!

### Motivation

I don't want to lose my place on the timeline, or I want to put it somewhere to remind me to read it after I've finished what I'm doing, etc.

**Comment by nemobis** [comment]
Agreed, this is one goal of https://github.com/mastodon/mastodon/issues/20827
